src/index.html

```diff
@@ -17,7 +17,7 @@
 
   </head>
   <body>
-    <div class="dev-warning">DEV MODE</div>
+    <!-- <div class="dev-warning">DEV MODE</div> -->
```

Comment:
If this stays like this, just remove it all together?

Reply:
It was committed by mistake, sorry!